composer.json

```diff
@@ -15,12 +15,18 @@
         "adodb/adodb-php": "5.20.2",
         "phpmailer/phpmailer": "5.2.16",
         "rospdf/pdf-php": "0.12.22",
-        "phpseclib/phpseclib": "2.0.2"
-        
+        "phpseclib/phpseclib": "2.0.2",
+        "zendframework/zendframework": "2.4.9",
+        "dino/dompdf-module": "dev-master",
+        "doctrine/couchdb": "@dev",
```

Comment:
Going off previously isntalled: dino/dompdf-module
https://github.com/openemr/openemr/blob/master/interface/modules/zend_modules/vendor/composer/installed.json#L248
and doctrine/couchdb
https://github.com/openemr/openemr/blob/master/interface/modules/zend_modules/vendor/composer/installed.json#L714

That's what was there.